lib/src/generator/generator_utils.dart

```diff
@@ -34,6 +34,16 @@ String generateCategoryJson(Iterable<Categorization> categories, bool pretty) {
   return encoder.convert(indexItems.toList());
 }
 
+String regEx(String? input){
+  if(input==null){
```

Comment:
It looks like this file is not formatted. Make sure to format with something like `dart format lib/`